fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt

```diff
@@ -73,6 +115,13 @@
     </field>
   </xsl:template>
 
+  <!-- the following template adds a utk_mods_ir_publication field -->
```

Comment:
@cdeaneGit, this is confusing.  You're adding code to your files that is already in institutional_repository. I'm pretty sure that means if we merge we'll have 2 of the same template.

Reply:
Looking at this again.